examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,72 @@
+"""
+Multi-subject connectome for ADHD
+=================================
+
+This example estimates a connectome from spheres in the brain. The coordinates
+of the spheres have been taken from a meta-analysis on neurosynth website.
+
+"""
+import matplotlib.pyplot as plt
+
+from nilearn import plotting
```

Comment:
As you are delaying the imports to where they are relevant (which I think is great), I think that you could move these two down to the plotting part.